.github/workflows/package.yml

```
@@ -2,40 +2,40 @@ name: Package
 
 on:
   push:
-    branches: [ "feature/version/upgrade"]
-    paths-ignore:
+     branches: [ "feature/version/upgrade"]
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 5

(indentation)